src/app/AppProvider.js

```diff
@@ -1,6 +1,9 @@
 import AppWrapper from './AppWrapper'
 
-export default {
+/**
+ * 
+ */ 
```

Comment:
☝️

Reply:
in preparation for documentation PRs later in the week ;)